### PR TITLE
[Refactor:TAGrading] Removing .html() from server.js

### PR DESCRIPTION
**Pull request by ziesski**
### Please check if the PR fulfills these requirements:

* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant
* [ ] Screenshots are attached to Github PR if visual/UI changes were made

### What is the current behavior?

Using `.html()` or `innerHTML` can introduce security vulnerabilities, especially related to Cross-Site Scripting (XSS) attacks, as they allow arbitrary HTML to be injected, potentially including malicious scripts.

### What is the new behavior?

It's safer to use `.text()` in jQuery or `.textContent` in plain JavaScript when setting or updating text content.

### Other information?
We will be covering our security standards in few of next week short presentation. This is one of the common mistakes that devs make. This PR is to replace some of `.html()` with `.text()` and reduce developers using outdated code as reference.

### CODE REVIEW

**Review by IDzyre — APPROVED** [review]
Seams good to me, I am not that good at jQuery, so it would be good to get at least one more review.

**Review by williamschen23 — DISMISSED** [review]
You change the color to red, but never change it back to the original color

**Comment by williamjallen** [comment]
@williamschen23 Do you have an example of faulty behavior related to the red text color?  At least from a brief look at the code, I don't see anything that looks immediately suspicious.

**Comment by williamschen23** [comment]
@ziesski

File: site/public/js/admin-gradeable-updates.js [hunk 1]
```diff
@@ -554,29 +554,29 @@ function ajaxUpdateGradeableProperty(gradeable_id, p_values, successCallback, er
...
-                        $('#save_status').html('Error Saving Changes');
+                        $('#save_status').text('Error Saving Changes').css('color', 'var(--text-black)');
                         setGradeableUpdateComplete();
                         console.error(`Failed to parse response from server: ${response}`);
                     },
                 });
             }
             catch (e) {
-                $('#save_status').html('Error Saving Changes');
+                $('#save_status').text('Error Saving Changes').css('color', 'var(--text-black)');
```
Comment (williamschen23) [inline site/public/js/admin-gradeable-updates.js:579]: same here

File: site/public/js/server.js [hunk 2]
```diff
@@ -1666,15 +1666,15 @@ function peerFeedbackUpload(grader_id, user_id, g_id, feedback) {
         contentType: false,
         success: function (data) {
             if (data.status === 'success') {
-                $('#save_status').html('All Changes Saved');
+                $('#save_status').text('All Changes Saved').css('color', 'var(--text-black)');
             }
             else {
-                $('#save_status').html('Error Saving Changes');
+                $('#save_status').text('Error Saving Changes');
```
Comment (williamschen23) [inline site/public/js/server.js:1672]: same here

**Comment by williamjallen** [comment]
Tagging @williamschen23 for further review as per in-person conversation.

**Review by williamschen23 — APPROVED** [review]
Tested some buttons for gradeable config, every color was correct

**Review by williamjallen — APPROVED** [review]
Code looks good.  @williamschen23 reviewed the functionality.  Good to merge.

**Pull request merged**